arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json

```diff
@@ -555,6 +555,72 @@
         }
       }
     },
+    "/webhdfs/v1/{flushFilePath}": {
```

Comment:
I agree. I am also removing this specific API since it isn't actually functional :).